pkg/server/container_create.go

```diff
@@ -79,12 +77,11 @@ func (c *criContainerdService) CreateContainer(ctx context.Context, r *runtime.C
 	// Prepare container image snapshot. For container, the image should have
 	// been pulled before creating the container, so do not ensure the image.
 	imageRef := config.GetImage().GetImage()
-	image, err := c.localResolve(ctx, imageRef)
+
+	// Ensure sandbox container image snapshot.
+	image, err := c.ensureImageExists(ctx, imageRef)
```

Comment:
In CRI, the assumption is that when create container, the image should have already been pulled, if not create should return error. So we could not use `ensureImageExists` in container create.

We use `ensureImageExists` in `RunPodSandbox`, because sandbox container image is runtime detail, CRI doesn't care about it.